assets/i18n/fr.json

```diff
@@ -25,25 +25,25 @@
     },
     "directions": {
         "dialog": {
-            "noOriginAddress": "Désolé, pouvez-vous répéter où vous souhaitez vous rendre ?",
+            "noOriginAddress": "Désolé, pouvez-vous répéter d'où vous partez ?",
             "noDestinationAddress": "Désolé, pouvez-vous répéter où vous souhaitez vous rendre ?",
             "noArrivalTime": "Désolé, pouvez-vous répéter à quelle heure vous souhaitez y être ?",
-            "noOriginAndDestinationAddresses": "Désolé, pouvez-vous répéter d'où vous souhaitez partir, et où vous souhaitez vous rendre ?",
+            "noOriginAndDestinationAddresses": "Désolé, pouvez-vous répéter d'où vous souhaitez partir et où vous souhaitez vous rendre ?",
             "noOriginAddressAndDepartureTime": "Désolé, pouvez-vous répéter d'où et à quelle heure vous souhaitez partir ?",
             "noOriginAddressAndArrivalTime": "Désolé, pouvez-vous répéter d'où vous souhaitez partir, ainsi que l'heure à laquelle vous souhaitez être à votre destination ?",
             "noDestinationAddressAndDepartureTime": "Désolé, pouvez-vous répéter où vous souhaitez vous rendre, ainsi que l'heure à laquelle vous comptez partir ?",
             "noDestinationAddressAndArrivalTime": "Désolé, pouvez-vous répéter où vous souhaitez vous rendre, ainsi que l'heure à laquelle vous souhaitez y être ?",
             "noDepartureTime": "Désolé, pouvez-vous répéter l'heure à laquelle vous souhaitez partir ?",
-            "sameLocations": "Super ! Il semble que vous soyez déjà là où vous souhaitez aller !",
+            "sameLocations": "Super ! Il semble que vous soyez déjà là où vous souhaitiez aller !",
             "noTripWithTravelMode": "Désolé, je n'ai pas de trajet pertinent à vous proposer {{ travel_mode }}."
         },
         "navigationTime": {
             "transit": "Se rendre de {{- location_from }} à {{- location_to }} en transports en commun prend habituellement {{ duration }}.",
             "bus": "Se rendre de {{- location_from }} à {{- location_to }} en bus vous prendra {{ duration }}.",
             "train": "Se rendre de {{- location_from }} à {{- location_to }} en train prend {{ duration }}.",
             "walking": "Marcher de {{- location_from }} jusqu'à {{- location_to }} vous prendra {{ duration }}.",
-            "bicycling": "Se rendre de {{- location_from }} à {{- location_to }} en vélo vous prendra {{ duration }}.",
-            "driving": "Conduire de {{- location_from }} jusqu'à {{- location_to }} prend habituellement {{ duration }}. Avec les conditions de circulation actuelles, cela vous prendra {{ duration_in_traffic }}.",
```

Comment:
This is actually not a mistake. Although this message should probably be split in two, the second part provides real time details in the case of a car journey and is necessary for the moment.

Reply:
thanks for the info!